dev/devicelab/lib/tasks/perf_tests.dart

```diff
@@ -1264,8 +1264,6 @@ class CompileTest {
 
   Future<TaskResult> run() async {
     return inDirectory<TaskResult>(testDirectory, () async {
-      final Device device = await devices.workingDevice;
-      await device.unlock();
```

Comment:
Do we also need to remove https://github.com/flutter/flutter/blob/master/dev/devicelab/bin/tasks/hello_world_ios__compile.dart#L10

Reply:
No, it's used to decide which device type to compile against.
https://github.com/flutter/flutter/blob/5bc9340233b5c8e5d52ae9ae6b72b54f4585cc4c/dev/devicelab/lib/tasks/perf_tests.dart#L1299

Reply:
Ah, I see.

BTW: are all perf tests not expected to have devices? I am wondering if any perf test needs a device and needs the `unlock` to avoid potential issues.

Reply:
I think a lot of perf tests are expected to have a device.  This change only impacts `CompileTest` though, which I don't think should ever need a device.
```dart
/// Measures how long it takes to compile a Flutter app and how big the compiled
/// code is.
class CompileTest {
```